src/GitHub.Exports/Settings/generated/IPackageSettings.cs

```diff
@@ -31,6 +31,7 @@ public interface IPackageSettings : INotifyPropertyChanged
         void Save();
         bool CollectMetrics { get; set; }
         bool EditorComments { get; set; }
+        bool EnableTraceLogging { get; set; }
```

Comment:
As the header suggests, this is a generated file so you should be editing `src/Common/settings.json` to modify this.

I don't really like this, but that's how it is for the moment :(

Reply:
I can't believe i missed that..